server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -66,6 +70,26 @@ public void upgradeMetadata(ServerContext context) {
     deleteExternalCompactions(context);
   }
 
+  private void removeCompactIds(ServerContext context) {
+    final String ZTABLE_COMPACT_ID = "/compact-id";
+    final String ZTABLE_COMPACT_CANCEL_ID = "/compact-cancel-id";
+
+    for (Entry<String,String> e : context.tableOperations().tableIdMap().entrySet()) {
+      final String tName = e.getKey();
+      final String tId = e.getValue();
+      final String zTablePath = Constants.ZROOT + "/" + context.getInstanceID().canonical()
+          + Constants.ZTABLES + "/" + tId;
+      try {
+        context.getZooReaderWriter().delete(zTablePath + ZTABLE_COMPACT_ID);
+        context.getZooReaderWriter().delete(zTablePath + ZTABLE_COMPACT_CANCEL_ID);
+      } catch (KeeperException | InterruptedException e1) {
+        throw new RuntimeException(
```

Comment:
Could this be IllegalStateException?

Reply:
Sure, it could be anything. I'm not sure that it matters though, right? Any exception is going to stop the upgrade process.

Reply:
I thought we we trying to migrate from throwing the generic Runtime exception as a principal.  

It does not matter much for this case (IllegalStateException is still a RunTimeException) - but it could make finding other places where we throw Runtime exceptions with something more appropriate easier to find.

Reply:
I changed it to `IllegalStateException`